cloud/google/clusteractuator_test.go

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
You need to fix this copyright now that we have the verify boilerplate script (it needs some blank lines).

Reply:
Fixed and added the copyright to the clientcomputeservice_test.go as well where it was missing.